LibraryItem.java

```diff
@@ -0,0 +1,37 @@
+abstract class LibraryItem{
+
```

Comment:
variables for the class are declared.
added an extra variable called "checkedout" to determine whether the required book is checked out.